packages/assets/generate.ts

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+const path = require('path');
+
+interface Entry {
+  name: string;
+  isDirectory: () => boolean;
+}
+
+const getFiles = async (entry: Entry) => {
+  const originalEntries = await fs.promises.readdir(entry, {
+    withFileTypes: true,
+  });
+
+  const files = await Promise.all(
+    originalEntries.map((foundEntry: Entry) => {
+      const res = path.resolve(entry, foundEntry.name);
+      const result = foundEntry.isDirectory() ? getFiles(res) : res;
+      return result;
+    }),
+  );
+
+  return Array.prototype.concat(...files);
+};
+
+const assets = async (orgId: string, destinationDirectory: string) => {
+  console.log(`Copying assets for ${orgId} to ${destinationDirectory}`);
+
+  const vaildOrgIds = ['atb', 'nfk'];
+
+  if (!vaildOrgIds.includes(orgId))
+    throw new Error(`Invalid orgId provided, valid orgIds are ${vaildOrgIds}`);
+
+  const commonFolder = path.join(__dirname, 'src', 'common');
```

Comment:
Tror kanskje dette må gjøres om, ettersom vi faktisk flytter til `lib/` ved kompillering fra node. Da vil __dirname ligge ett nivå opp. Så forslag:

```suggestion
  const commonFolder = path.join(__dirname, '..', 'src', 'common');
```

Reply:
Mulig det er hensiktsmessig å flytte source filer (ts) slik at det blir likt med bruk av ts-node og kjøring av kompilerte. Altså at vi flytter generate.ts og bin.ts til `src/`. Kanskje det da gir mest mening å flytte assets filene til en `static/` mappe heller f.eks. Føler `src/` til de filene er litt misvisende og?

Reply:
Jepp, det er i boks nå. Assetsfilene er nå i `files/`, mens  `generate.ts` og `bin.ts` lever i `src/`